adapters/taboola/taboola.go

```diff
@@ -35,9 +39,17 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		return nil, []error{err}
 	}
 
+	requestEndpoint := a.endpoint
+	if len(request.Imp) > 0 {
+		if request.Imp[0].Native != nil {
+			requestEndpoint = NATIVE_ENDPOINT
+		}
+
+	}
+
 	requestData := &adapters.RequestData{
 		Method: "POST",
-		Uri:    a.endpoint + "/" + taboolaRequest.Site.ID,
+		Uri:    requestEndpoint + "/" + taboolaRequest.Site.ID,
```

Comment:
This can also be a part of "buildEndpointUrl" function

Reply:
done